react-client/src/components/Shipping.jsx

```diff
@@ -1,8 +1,18 @@
 import React from 'react';
 
-const Shipping = ({time, unitsSold, shipping}) => {
+const Shipping = ({shipping}) => {
+
+  const getTimeDiff = (diff) => {
+    let days = Math.floor(diff / 86400)

```

Comment:
it's not necessary at all here, but stylistically I like declaring constants like these time values in ALL_CAPS.  This could be MS_PER_DAY for instance

Reply:
I like it, will add it to my refactoring todo list.